doc/modules/clustering.rst

```diff
@@ -498,6 +498,15 @@ geometrical shape.
 |face_kmeans|                          |face_discretize|
 =====================================  =====================================
 
+Spectral Clustering Graphs
+--------------------------
+
+Spectral Clustering can also be used to cluster graphs by their spectral embeddings.
```

Comment:
On the basis of this description, the same may be true of affinity propagation.

I think though that what was being sought in the issue is an example in `examples/` that illustrates this application.

Reply:
This would have been enough for me, but I'm happy to add an example to `examples/` if you think that's appropriate. Let me know =)

Reply:
I'm also not sure what you mean by 

> On the basis of this description, the same may be true of affinity propagation.

Reply:
I just mean that if all you want in the documentation is to say that the adjacency matrix can be used as a precomputed affinity matrix, that's true of anywhere else `affinity='precomputed'` is supported (ignoring hierarchical clustering where `affinity` is used for a distance metric).

Reply:
I hadn't realized -- that's a powerful insight. 

Here's my point: for folks like myself, we have graphs, and we don't think of them as "precomputed affinity matrices" which is why I had trouble writing two lines of code. I don't think the code needs much adjusting ([though I'm advocating adding this as a method to networkx as well](https://github.com/networkx/networkx/pull/2682)) I just think there need to be slightly more explicit docs here. 
- If you'd like me to add a snippet in the docs of other methods where graphs can be used as precomputed affinity matrices, I'm happy to do that. 
- I'm also happy to add something in the examples folder, but I would still ask you keep this snippet here.